README.md

```diff
@@ -35,8 +35,8 @@ Simply retrieve location over time. With custom manager settings
 ReactiveLocation.singleLocationProducer { manager in
 	manager.distanceFilter = 1000
 	manager.desiredAccuracy = kCLLocationAccuracyBest
-}.startWithNext { location in
-    print(location)            
+}.startWithNext { [unowned self] location in
```

Comment:
Master branch is Swift 3 & ReactiveSwift so we should provide example in new syntax `startWithValues`.